src/components/ViewAgents/index.tsx

```diff
@@ -8,8 +8,8 @@ function CreateWizards() {
 
   return (
     <div className="w-100">
-      <MyWizards />
-      <PopularWizards />
+      <HomePageWizardPlaceholder />
```

Comment:
Changes name to `WizardPlaceholder`?

Reply:
Done https://github.com/elna-ai/ELNA-DApp/pull/215/commits/83aca427d12d7921614d598cad9c8d616f574896